.github/workflows/ci.yml

```diff
@@ -35,10 +35,23 @@ jobs:
       - name: Check formatting
         run: cargo xtask check fmt -v
 
+  typos:
+    name: Check typos
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v3
+
```

Comment:
Suggestion: let’s cache the binaries, as done in other jobs:
```yml
      - name: Binary cache
        uses: actions/cache@v3
        with:
          path: ./.cargo/local_root/bin
          key: ${{ runner.os }}-bin-${{ github.job }}-${{ hashFiles('xtask/src/bin_version.rs') }}
```

Reply:
I am all over the place today 🤦‍♂️